### PR TITLE
Fixed a bug in YoloNASPose.export() that prevented to export model for BS>1

### PR DESCRIPTION
**Pull request by BloodAxe**

### CODE REVIEW

File: src/super_gradients/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = "3.3.0"
+__version__ = "3.3.1"
```
Comment (Louis-Dupont) [inline src/super_gradients/__init__.py:1]: Version change because you want to release a fix already right ?

**Review by Louis-Dupont — COMMENTED** [review]
LGTM

File: src/super_gradients/__init__.py (hunk 1 above)
Comment (BloodAxe) [reply src/super_gradients/__init__.py:1]: Up!

**Review by Louis-Dupont — APPROVED** [review]
LGTM

**Pull request merged**